SentinelOne/cloud_funnel2.0/ingest/parser.yml

```
@@ -581,6 +581,8 @@ stages:
           action.properties.TargetDomainName: '{{json_event.message.get("event.login.accountDomain")}}'
           action.properties.TargetUserSid: '{{json_event.message.get("event.login.accountSid")}}'
           action.properties.IpAddress: '{{json_event.message.get("src.endpoint.ip.address")}}'
+          user.target.name: '{{json_event.message.get("event.login.userName")}}'
+          user.target.domain: '{{json_event.message.get("event.login.accountDomain")}}'
```

In this format, we can also define `user.target.id`:
```suggestion
          user.target.domain: '{{json_event.message.get("event.login.accountDomain")}}'
          user.target.id: '{{json_event.message.get("event.login.accountSid")}}'
```